en/config_heli/README.md

```diff
@@ -1,31 +1,29 @@
 # Helicopter Configuration
 
-This section contains topics related to helicopter configuration and tuning.
-
-:::note
-Helicopters are less well-supported on than other types of vehicles.
-For example, PX4 does not support helicopters with coaxial or dual rotor types, and features such as RPM governor and autorotation.
-We would welcome your [contribution](../contribute/README.md) of new features, new frame configurations, or other improvements.
-:::
+This section contains topics related to [helicopter](../frames_helicopter/README.md) configuration and tuning.
 
 ## Supported Configurations
 
 Supported helicopter configurations:
 
 - Single main rotor with swash-plate controlled by up to 4 swash-plate servos and a mechanically uncoupled tail rotor driven by an ESC.
+- Mechanically coupled tails controlled by a servo can be flown by mapping the tail servo to the tail motor output and setting an adequate range and disarmed value for the servo.
```

Comment:
It's a separate `CA_AIRFRAME` type which gives you a tail servo

Reply:
Thanks @bkueng : 

10: Helicopter (tail ESC)
11: Helicopter (tail Servo)

But how do you select the frames? I.e. there is only one airframe shown for Helicopter in http://docs.px4.io/main/en/airframes/airframe_reference.html#helicopter

Reply:
Yes there's currently only 1 helicopter airframe. We could add a generic one for tail servo's as well, but I'm not sure if it's worth it at this point.

Reply:
@bkueng Just to be very clear, to access the tail servo option they would select the generic helicopter airframe then manually modify the CA_AIRFRAME to 11? A that point it will work as a tail servo? Will the actuator UI change at all?

Reply:
Correct, yes the UI will reflect that.